Library/brew.sh

```diff
@@ -70,6 +70,10 @@ then
     HOMEBREW_RUBY_PATH="/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby"
   else
     HOMEBREW_RUBY_PATH="$(which ruby)"
+    if [[ -z "$HOMEBREW_RUBY_PATH" ]]
```

Comment:
@sjackman Nice idea. I'll leave improving stuff like this to you because you've got more ideas of the audience. Just making it super generic for now.